ruby_connectors/lib/ruby_connectors/github_connector/cataloger.rb

```
@@ -3,7 +3,7 @@
 module RubyConnectors
   module GithubConnector
     class Cataloger
-      def catalog
+      def catalog(activity)
```

RuboCop (1.73)

[warning] Unused method argument - `activity`. If it's necessary, use `_` or `_activity` as an argument name to indicate that it won't be used. If it's unnecessary, remove it. You can also write as `catalog(*)` if you want the method to accept any arguments but don't care about them.

(Lint/UnusedMethodArgument)